Index: DataService/Objects/Rating.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>namespace DataService.Objects\r\n{\r\n    public class Rating\r\n    {\r\n        public int Id { get; set; }\r\n        public int User_Id { get; set; }\r\n        public int Title_Id { get; set; } \r\n        public float rating { get; set; }\r\n        \r\n        public Users Users { get; set; }\r\n\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- DataService/Objects/Rating.cs	(revision dbc6e47a643bd01d704a59fd7cc5b845e45b9133)
+++ DataService/Objects/Rating.cs	(date 1604573126945)
@@ -7,7 +7,5 @@
         public int Title_Id { get; set; } 
         public float rating { get; set; }
         
-        public Users Users { get; set; }
-
     }
 }
\ No newline at end of file
Index: DataService/Services/TitleDataService.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>using DataService.Objects;\r\n\r\nnamespace DataService.Services\r\n{\r\n    public class TitleDataService\r\n    {\r\n        public TitleDataService()\r\n        {\r\n            using var ctx = new ImdbContext();\r\n        }\r\n\r\n        public Genre GetGenre(int id)\r\n        {\r\n            using var ctx = new ImdbContext();\r\n            return ctx.genre.Find(id);\r\n        }\r\n        \r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- DataService/Services/TitleDataService.cs	(revision dbc6e47a643bd01d704a59fd7cc5b845e45b9133)
+++ DataService/Services/TitleDataService.cs	(date 1604573015858)
@@ -14,6 +14,5 @@
             using var ctx = new ImdbContext();
             return ctx.genre.Find(id);
         }
-        
     }
 }
\ No newline at end of file
Index: UnitTest/TitleDataServiceTest.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>using System;\r\nusing System.Linq;\r\nusing Xunit;\r\n\r\n\r\nnamespace PortFolio2.Tests\r\n{\r\n    \r\n    //nik\r\n    public class TitleDataServiceTest\r\n    {\r\n        [Fact]\r\n        public void GetGenre()\r\n        {\r\n            var service = new DataService.Services.TitleDataService();\r\n            var genre = service.GetGenre(1);\r\n            Assert.Equal(1, genre.Id);\r\n            Assert.Equal(\"Sport\", genre.Name);\r\n        }\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- UnitTest/TitleDataServiceTest.cs	(revision dbc6e47a643bd01d704a59fd7cc5b845e45b9133)
+++ UnitTest/TitleDataServiceTest.cs	(date 1604573096191)
@@ -1,6 +1,7 @@
 using System;
 using System.Linq;
 using Xunit;
+using DataService.Services;
 
 
 namespace PortFolio2.Tests
Index: DataService/PostgresSQL_Connect_String.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>namespace DataService\r\n{\r\n    public class PostgresSQL_Connect_String\r\n    {\r\n        private string host = \"localhost\";\r\n        private string db = \"imdb\";\r\n        private string UserId = \"postgres\";\r\n        private string pwd = \"Flintholm2020\";\r\n\r\n        public override string ToString()\r\n        {\r\n            return $\"host={host};db={db};uid={UserId};pwd={pwd}\";\r\n        }\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- DataService/PostgresSQL_Connect_String.cs	(revision dbc6e47a643bd01d704a59fd7cc5b845e45b9133)
+++ DataService/PostgresSQL_Connect_String.cs	(date 1604573213203)
@@ -5,7 +5,7 @@
         private string host = "localhost";
         private string db = "imdb";
         private string UserId = "postgres";
-        private string pwd = "Flintholm2020";
+        private string pwd = "Nvp92agn";
 
         public override string ToString()
         {
